Functions/xASL_bids_MergeNifti_Delete.m

```diff
@@ -0,0 +1,33 @@
+function xASL_bids_MergeNifti_Delete(NiftiPaths)
```

Comment:
@jan-petr  This function already exists: `xASL_adm_DeleteFilePair` so can be removed